lib/charms/ovn_charm.py

```diff
@@ -683,3 +679,26 @@ def __init__(self, **kwargs):
                 '/etc/neutron/neutron_ovn_metadata_agent.ini': [
                     metadata_agent],
             })
+
+
+class BaseVictoriaOVNChassisCharm(BaseOVNChassisCharm):
+    """Victoria incarnation of the OVN Chassis base charm class."""
+    abstract_class = True
+
+    def __init__(self, **kwargs):
+        """Allow augmenting behaviour on external factors."""
+        super().__init__(**kwargs)
+        self.nrpe_check_services = [
+            'ovn-controller',
+            'ovs-vswitchd',
+            'ovsdb-server',
+        ]
+        if self.enable_openstack:
+            metadata_agent = 'neutron-ovn-metadata-agent'
+            self.nrpe_check_services.append(metadata_agent)
+            self.packages.extend([metadata_agent])
+            self.services.append(metadata_agent)
+            self.restart_map.update({
+                '/etc/neutron/neutron_ovn_metadata_agent.ini': [
+                    metadata_agent],
+            })
```

Comment:
You don't need to add this unless it is different from the previous version. charms.openstack will automatically pick up the newest one when none exists for the current version.

Reply:
Thanks, you mean then I don't need this either? https://review.opendev.org/#/c/761584/2/src/lib/charm/openstack/ovn_chassis.py

Reply:
It is not required from charms.openstack's pov. It may be that I have (ab)used the `release` class variable in the code so that you need to do it, but if that is the only reason we keep adding them we should rework the code to use a variable/property produced at runtime with accurate release information instead of relying on manual copy/pasta for each release.

Reply:
Otoh, I think the code only ever refers to the `train` release specifically, because that was different, so perhaps we don't need to care about it at all?